Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -649,6 +649,19 @@ final class UIViewExtensionsTests: XCTestCase {
         // simple empty case test
         XCTAssertNil(container.widthConstraint)
     }
+
+    func testRemoveBlur() {
+        let view = UIView()
+        let blurEffect = UIBlurEffect(style: .prominent)
+        let blurEffectView = UIVisualEffectView(effect: blurEffect)
+        blurEffectView.frame = view.bounds
+        view.addSubview(blurEffectView)
+        view.removeBlur()
+        XCTAssertTrue(view.subviews.contains(blurEffectView))
```

Comment:
That test should fail, as you just removed it.
You can take the test from `testBlur()`
```suggestion
        view.addSubview(blurEffectView)

        XCTAssert(view.subviews.first is UIVisualEffectView)

        view.removeBlur()
```

Reply:
I have made modifications.